gocloud/gocloud.go

```diff
@@ -0,0 +1,42 @@
+package gocloud
+
+import (
+	"errors"
+	"fmt"
+	"github.com/scorelab/gocloud/aws"
+	"github.com/scorelab/gocloud/google"
+)
+
+//gocloud is a interface which hide differece between different cloud providers
+
+type Gocloud interface {
+	Createnode(request interface{}) (resp interface{}, err error)
+	Startnode(request interface{}) (resp interface{}, err error)
+	Stopnode(request interface{}) (resp interface{}, err error)
+	Deletenode(request interface{}) (resp interface{}, err error)
+	Rebootnode(request interface{}) (resp interface{}, err error)
+}
+
+const (
+	Amazonprovider = "aws"
+	Googleprovider = "google"
+	Secretkey      = "dummy"
+	Secretid       = "dummy"
```

Comment:
replace `dummy` with `SECRET_ID`

I believe this will be replaced with configuration reader?